Resources/config/doctrine/model/RefreshToken.orm.xml

```diff
@@ -12,6 +12,11 @@
         </id>
         <field name="expiry" type="datetime_immutable" />
         <field name="revoked" type="boolean" />
+        <indexes>
+            <index name="idx_oauth2_refresh_token_access_token" columns="access_token"/>
```

Comment:
@franjo-zadelj-trikoder this is redundant, doctrine relationships already get an index by default